quic/s2n-quic-transport/src/ack/mod.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod interest;
```

Comment:
can you also re-export core here?

```suggestion
pub use s2n_quic_core::ack::*;
pub mod interest;
```